src/soc_ifc/rtl/soc_ifc_subsystem_reg.rdl

```diff
@@ -0,0 +1,160 @@
+    desc = "Subsystem SoC Interface Base Address Upper 32 bits (from AXI)
+            [br]Caliptra Access: RO
+            [br]SOC Access:      RO";
+    strap addr_h[32];
+} SS_SOC_IFC_BASE_ADDR_H;
+
+reg {
+    name = "Subsystem MCI Base Address (Low)";
+    desc = "Subsystem MCI Base Address Lower 32 bits (from AXI)
+            [br]Caliptra Access: RO
+            [br]SOC Access:      RO";
+    strap addr_l[32];
+} SS_MCI_BASE_ADDR_L;
+
+reg {
+    name = "Subsystem MCI Base Address (High)";
+    desc = "Subsystem MCI Base Address Upper 32 bits (from AXI)
+            [br]Caliptra Access: RO
+            [br]SOC Access:      RO";
+    strap addr_h[32];
+} SS_MCI_BASE_ADDR_H;
+
+reg {
+    name = "Subsystem Recovery Interface Base Address (Low)";
+    desc = "Subsystem Recovery Interface Base Address Lower 32 bits (from AXI)
+            [br]Caliptra Access: RO
+            [br]SOC Access:      RO";
+    strap addr_l[32];
+} SS_RECOVERY_IFC_BASE_ADDR_L;
+
+reg {
+    name = "Subsystem Recovery Interface Base Address (High)";
+    desc = "Subsystem Recovery Interface Base Address Upper 32 bits (from AXI)
+            [br]Caliptra Access: RO
+            [br]SOC Access:      RO";
+    strap addr_h[32];
+} SS_RECOVERY_IFC_BASE_ADDR_H;
+
+reg {
+    name = "Subsystem Fuse Controller Base Address (Low)";
+    desc = "Subsystem Fuse Controller Base Address Lower 32 bits (from AXI)
+            [br]Caliptra Access: RO
+            [br]SOC Access:      RO";
+    strap addr_l[32];
+} SS_OTP_FC_BASE_ADDR_L;
+
+reg {
+    name = "Subsystem Fuse Controller Base Address (High)";
+    desc = "Subsystem Fuse Controller Base Address Upper 32 bits (from AXI)
+            [br]Caliptra Access: RO
+            [br]SOC Access:      RO";
+    strap addr_h[32];
+} SS_OTP_FC_BASE_ADDR_H;
+
+reg {
+    name = "Subsystem UDS Seed Destination Base Address (Low)";
+    desc = "Subsystem UDS Seed Destination Base Address Lower 32 bits (from AXI). Used as the write address when programming the UDS seed during manufacturing.
+            [br]Caliptra Access: RO
+            [br]SOC Access:      RO";
+    strap addr_l[32];
+} SS_UDS_SEED_BASE_ADDR_L;
+
+reg {
+    name = "Subsystem UDS Seed Destination Base Address (High)";
+    desc = "Subsystem UDS Seed Destination Base Address Upper 32 bits (from AXI). Used as the write address when programming the UDS seed during manufacturing.
+            [br]Caliptra Access: RO
+            [br]SOC Access:      RO";
+    strap addr_h[32];
+} SS_UDS_SEED_BASE_ADDR_H;
+
+reg {
+    name = "Subsystem SoC Nonce";
+    desc = "TODO Need a register that needs a fixed NONCE (this is a register (not a fuse from Caliptra POV) that MCU can write)"; // TODO
+    field { sw=rw; hw=r; resetsignal = cptra_rst_b; } nonce[32]=0;
+} SS_SOC_NONCE[16] @0x560;
+
+reg {
+    name = "DEBUG INTENT";
+    desc = "Strap from SoC in Subsystem configuration that indicates if this boot cycle will be used for debug.
```

Comment:
We shld say that this strap is sticky?

Reply:
Also should we make a note in integration spec that passive mode SHALL tie this to '0 or does active mode == 0 already overrides this?

Reply:
Updated Integ spec in #640